cmd/sge.go

```diff
@@ -375,10 +375,24 @@ func generateBabylonScript(fileTemplate string, l NonMemModel) ([]byte, error) {
 		Command          string
 	}
 
+	commandComponents := []string{
```

Comment:
No logical change here, just generating the command from string slices to make it cleaner.